_security-plugin/access-control/multi-tenancy.md

```diff
@@ -37,15 +37,15 @@ config:
     opensearch-dashboards:
       multitenancy_enabled: true
       server_username: opensearch
-      index: '.opensearch-dashboards'
+      index: '.kibana'
     do_not_fail_on_forbidden: false
 ```
 
 Setting | Description
-:--- | :---
+:--- | :--- 
```

Comment:
I added a space there so there was a change to commit.  Wasn't sure how to commit with no changes.